include/stout/flags/flags.cc

```diff
@@ -24,6 +24,8 @@ void Parser::AddFieldsAndSubcommandsOrExit(google::protobuf::Message& message) {
   for (int i = 0; i < descriptor->field_count(); i++) {
```

Comment:
Maybe add a comment explaining that this iterates over fields in *declaration* order (since that's important for positional argument processing)